src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -1022,7 +1025,7 @@ public bool TryGetNodeInfo(string name, out CustomNodeInfo info)
                     Enumerable.Empty<NoteModel>(),
                     0,
                     0,
-                    newId, string.Empty);
+                    newId, currentWorkspace.ElementResolver, string.Empty);
 
```

Comment:
Again in this case, where a new `CustomNodeWorkspaceModel` is being created by collapsing nodes in an existing workspace, the `ElementResolver` is simply assigned to that of the parent workspace.